seclang/rule_parser.go

```diff
@@ -342,17 +342,15 @@ func ParseRule(options RuleOptions) (*coraza.Rule, error) {
 	actions := ""
 
 	if options.WithOperator {
-		matches := ruleTokenRegex.FindAllString(options.Data, -1)
+		matches := ruleTokenRegex.FindAllString(options.Data, 3) // we use at most second match
```

Comment:
Please do confirm @jptosso @fzipi

Reply:
Yes, 1 matches the whole thing, plus the first operator block, then actions block.